README.md

```diff
@@ -162,7 +194,35 @@ To cite Concrete in academic papers, please use the following entry:
   note={\url{https://github.com/zama-ai/concrete}},
 }
 ```
+### 🟨 Contributing 
+
+There are two ways to contribute to Concrete. You can:
+- [Open issues](https://github.com/zama-ai/concrete/issues/new/choose) to report bugs and typos, or to suggest new ideas
+- Request to become an official contributor by emailing hello@zama.ai.
+
+Becoming an approved contributor involves signing our Contributor License Agreement (CLA)). Only approved contributors can send pull requests (PRs), so get in touch before you do!
+<br></br>
```

Comment:
or, here, you move the part "If you have built awesome projects using Concrete, feel free to let us know and we'll link to it." or an equivalent?

Reply:
OK thx. But do we have a link to  https://docs.zama.ai/concrete/developer/contributing here?